S/SBML/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env julia
+using BinaryBuilder, Pkg
+
+name = "SBML"
+version = v"5.19.0"
+sources = [
+        ArchiveSource(
+          "https://github.com/sbmlteam/libsbml/archive/v5.19.0.tar.gz",
+          "127a44cc8352f998943bb0b91aaf4961604662541b701c993e0efd9bece5dfa8"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir
+SBMLNAME=libsbml-*
+
+cd ${WORKSPACE}/srcdir/${SBMLNAME}
+mkdir build
+cd build
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${prefix} \
+  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+  -DCMAKE_LIBRARY_PATH=${prefix}/lib \
+  -DCMAKE_INCLUDE_PATH="${prefix}/include/libxml2;${prefix}/include" \
```

Comment:
Better to use more specific [environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables)
```suggestion
  -DCMAKE_LIBRARY_PATH=${libdir} \
  -DCMAKE_INCLUDE_PATH="${includedir}/libxml2;${includedir}" \
```

Reply:
:+1: